dht/sharder.hh

```diff
@@ -77,6 +132,7 @@ class selective_token_range_sharder {
     dht::token _start_token;
     std::optional<range_bound<dht::token>> _start_boundary;
 public:
+    // Initializes the selective_token_range_sharder with a token range and shard_id of interest.
```

Comment:
The comment just repeats the code so it would probably be better to just drop it.